src/app/modules/wallet/pages/account-list/account-list.component.ts

```diff
@@ -30,12 +33,19 @@ export class AccountListComponent implements OnInit {
   @ViewChild(MatPaginator, {static: true}) paginator: MatPaginator;
   @ViewChild(MatSort, {static: true}) sort: MatSort;
 
-  constructor() {
+  constructor(private walletService: WalletService) {
     // Create 100 users.
-    const users = Array.from({length: 100}, (_, k) => createNewUser(k + 1));
+    //const users = Array.from({length: 100}, (_, k) => createNewUser(k + 1));
 
     // Assign the data to the data source for the table to render.
-    this.dataSource = new MatTableDataSource(users);
+
+    walletService.accounts$.pipe(
```

Comment:
This subscription is not cleaned up and will memory leak. Let's use `takeUntil(this.destroyed$)` as a pattern we use in other similar components: https://github.com/prysmaticlabs/prysm-web-ui/blob/c0ad24bb456fc43ddd5ca563878780c6c3330bda/src/app/modules/auth/login/login.component.ts#L63